### DISCUSSION TITLE
Please add a RSS feed to the blog

### DISCUSSION THREAD

**Discussion by kawpii** [medusajs/medusa#1721] [Feature Requests]
There are users who use RSS feeds over newsletters like me, so please add a RSS feed to the blog

**Comment by olivermrbl** [comment]
@shahednasser Do you happen to know, what this would require?

**Reply by shahednasser** [reply]
I think this is possible with a Next.js plugin. I can look into it

**Reply by kawpii** [reply]
https://dev.to/j471n/how-to-add-rss-feed-in-nextjs-blog-34j1

**Comment by reev-create** [comment]
I think that would require data fetch via socket which would slow down the server once traffic increases.
